sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -57,7 +57,9 @@ export interface TracingClient {
         tracingOptions?: OperationTracingOptions;
     }>(name: string, operationOptions?: Options, spanOptions?: TracingSpanOptions): {
         span: TracingSpan;
-        updatedOptions: Options;
+        updatedOptions: Options & {
```

Comment:
I don't _think_ this is a breaking change right? We can narrow return types and expand inputs I think? Just checking...

I was also wondering if we want to name this type - what do you think?

Reply:
Since we're providing more accurate typing it's not breaking, but it does save the consumer from needing to typeguard. We *know* we will always set the context, so might as well express that.

I'm totally down for providing a nicer alias than the literal - any names come to mind?

Reply:
WithTracingOptions? OptionsWithTracing? I'll comment here if I have any better ideas but for now that's all I can come up with 😄

Reply:
I tweaked this slightly since I wasn't sure if we wanted to make *all* tracing options required (as there's only one today) and went with `OptionsWithTracingContext` - I'll push the update in a bit